ui/basicTab/basicTab.html

```diff
@@ -0,0 +1,183 @@
+<!-- Basic section -->
```

Comment:
These are just split from the main html file.
No functional changes, since `main.html` is effectively unchanged.